generic/src/test/java/org/datahem/processor/generic/GenericStreamPipelineTest.java

```diff
@@ -0,0 +1,600 @@
+            .put("repeatedFloat", new JSONArray().put(1).put(2).put(3))
+            .put("repeatedBool", new JSONArray().put("true").put("false").put("true"))
+            .put("repeatedBytes", new JSONArray()
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes()))
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes()))
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes())))
+            .put("repeatedEnum", new JSONArray().put(0).put(1).put(2));
+        
+        JSONObject messageChild = new JSONObject(testPayloadObject.toString());
+        messageChild.remove("_ATTRIBUTES");
+        messageChild.put("stringMap", new JSONObject()
+            .put("timestamp", "2013-08-16T23:36:32.444Z")
+            .put("uuid", "123-456-abc")
+            .put("source", "test"));
+
+        testPayloadObject
+            .put("messageChild", messageChild)
+            .put("repeatedMessage", new JSONArray()
+                .put(messageChild)
+                .put(messageChild));
+
+        String testPayload = testPayloadObject.toString();
+        byte[] payload = testPayload.getBytes(StandardCharsets.UTF_8);
+        PubsubMessage pm = new PubsubMessage(payload, attributes);
+
+        List<TableRow> attributes = new ArrayList<TableRow>();
+        attributes.add(new TableRow().set("key", "source").set("value", "test"));
+        attributes.add(new TableRow().set("key", "uuid").set("value", "123-456-abc"));
+        attributes.add(new TableRow().set("key", "timestamp").set("value", "2013-08-16T23:36:32.444Z"));
+
+        TableRow childMessage = new TableRow()
+                .set("StringField", "a string")
+                .set("Int32Field", 32)
+                .set("Int64Field", 64)
+                .set("DoubleField", 1.1)
+                .set("FloatField", 1.0)
+                .set("BoolField", true)
+                .set("BytesField", "Ynl0ZXM=")
+                .set("EnumField", 1)
+                .set("repeatedString", Stream.of("one", "two", "three").collect(Collectors.toList()))
+                .set("repeatedInt32", Stream.of(32, 64, 128).collect(Collectors.toList()))
+                .set("repeatedInt64", Stream.of(64, 128, 256).collect(Collectors.toList()))
+                .set("repeatedDouble", Stream.of(1.1, 1.2, 1.3).collect(Collectors.toList()))
+                .set("repeatedFloat", Stream.of(1.0, 2.0, 3.0).collect(Collectors.toList()))
+                .set("repeatedBool", Stream.of(true, false, true).collect(Collectors.toList()))
+                .set("repeatedBytes", Stream.of("Ynl0ZXM=", "Ynl0ZXM=", "Ynl0ZXM=").collect(Collectors.toList()))
+                .set("repeatedEnum", Stream.of(0,1,2).collect(Collectors.toList()))
+                .set("stringMap", attributes);
+
+        TableRow assertTableRow = new TableRow()
+            .set("StringField", "a string")
+            .set("Int32Field", 32)
+            .set("Int64Field", 64)
```

Comment:
Why not a real int64 like 79239016081? So that we really make sure it's not limited to 32 bits...